src/resolvers/index.ts

```diff
@@ -22,10 +23,27 @@ const noPerms = {
 
 export const resolvers: Resolvers<Context> = {
   Query: {
-    me: (_, _params, ctx) => {
+    me: async (_, _params, ctx) => {
+      let username;
       if (!ctx.session.user) return null;
-      console.log(ctx.session.user.username);
-      return null;
+      else if (!ctx.session.user.username) return null;
+      else username = ctx.session.user.username;
+
+      return await getMe(username);
+    },
+    findUserById: async (_, _params, ctx) => {
+      if (!ctx.session.user) return null;
+      else if (!_params.id) return null;
+      else return await getUserByID(+_params.id);
+    },
+    courses: async (_, _params, ctx) => {
+      if (!ctx.session.user) return null;
+      else if (!_params.term) return null;
+      else return [await getCourses(_params.term)];
```

Comment:
This is a bit confusing given `getCourses` is implying a plural (array) of response and there's something odd happening by creating an array here.